backend/project_name/__init__.py

```diff
@@ -0,0 +1,3 @@
+from __future__ import absolute_import
```

Comment:
I don't think we need __future__ as we're not Python 2 compatible. Please remove this elsewhere.